providers/queueing/memoryQueue.js

```diff
@@ -33,14 +34,19 @@ class MemoryQueue {
     const message = this.data[0]
     if (!message) return null
     this.data[0].dequeueCount++
-    if (message.dequeueCount <= 5) return Promise.resolve({ original: message, data: JSON.parse(message.messageText) })
+    if (message.dequeueCount <= 5) return Promise.resolve({ original: message, data: this._parseData(message) })
```

Comment:
This was the `messageText` property of the `message` object before. Was this intentional?

Reply:
`message` is destructured to extract the `messageText` property on line 42